tests/network/setup_multimachine.pm

```diff
@@ -54,16 +52,14 @@ sub run {
                 assert_script_run 'systemctl restart  wicked';
             }
             else {
-                assert_script_run 'systemctl stop NetworkManager';
-                assert_script_run 'systemctl disable NetworkManager';
+                disable_and_stop_service('NetworkManager');
                 assert_script_run 'systemctl enable wicked';
                 assert_script_run 'systemctl start  wicked';
             }
         }
         #Opensuse versions
         if (is_opensuse) {
-            assert_script_run 'systemctl stop NetworkManager';
-            assert_script_run 'systemctl disable NetworkManager';
+            disable_and_stop_service('NetworkManager');
```

Comment:
Using `disable_and_stop_service` is fine, but how will this fix issue with not installed NetworkManager on TW ?
Condition has to be updated for TW when NetworkManager is not installed.

Reply:
@dzedro thanks, correct. I wrongly interpret the code in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/Utils/Systemd.pm.
@pdostal @dzedro I'm thinking about adding detection via `systemctl is-active` and `systemctl is-enabled` to `disable_and_stop_service()` so users of it can be lazy. Any opinion? Should it be just optional? Or the default (which is different that how it was used, thus can break something), but possible to disable? Or was this is meant by `ignore_failure`? I wonder if there can be other failures that requesting to stop and/or disable service which belongs to not-installed package.

Reply:
One of `systemctl is-active` or `systemctl is-enabled` should be enough to determine `NetworkManager` being installed

Reply:
OK, I'll use one of them in `setup_multimachine.pm`. I just wondered whether wouldn't be better using both of them in `disable_and_stop_service()` and remove `ignore_failure` (or, put it different way: reconsider `disable_and_stop_service()` before doing fixes outside of it).

Reply:
> I just wondered whether wouldn't be better using both of them in `disable_and_stop_service()` and remove `ignore_failure`

That way it would be better